pro/src/apiClient/adage/models/AcademyEnum.ts

```diff
@@ -0,0 +1,40 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+/**
+ * An enumeration.
+ */
+export enum AcademyEnum {
+  AMIENS = 'Amiens',
+  AIX_MARSEILLE = 'Aix-Marseille',
+  BESAN_ON = 'Besançon',
```

Comment:
(question) Je suppose que c'est l'outil qui a généré ça, mais on pourrait sûrement remplacer les caractères accentués par leur version "non accentuée", plutôt que par des `_` ?

```suggestion
  BESANCON = 'Besançon',
```

idem plus bas.

Reply:
J'ai supprimé la partie `enum` dans le modèle _pydantic_ de réponse. L'intérêt principal étant de générer le code front, s'il est inutilisable, autant aller au plus simple.